Resources/Prototypes/Entities/Structures/Walls/walls.yml

```
@@ -743,6 +745,8 @@
         acts: ["Destruction"]
     destroySound:
       path: /Audio/Effects/metalbreak.ogg
+  - type: Reflect
+    reflectProb: 0.1
```

```suggestion
    reflectProb: 1
```

Shuttle walls should reflect 100% of the time.